_assets/stylesheets/_card.scss

```diff
@@ -0,0 +1,3 @@
+.card .card-clear { border-style: none; }
+.card-img-circle { border-radius: 50%; }
```

Comment:
We should talk about where these next two lines should go, because I don't think they're limited to `.card-clear`.